CHANGELOG.md

```diff
@@ -11,7 +11,7 @@
 - The Reader callback events' names have changed.
     - `On{EVENT_NAME}` is now `On{EVENT_NAME}Event`.
     - `{FIELD_NAME}Updated` is now `On{FIELD_NAME}Update`.
-- The generated Writers have been renamed, previously they were called. `{COMPONENT_NAME}.Requirable.Writer` to `{COMPONENT_NAME}Writer`.
+- The generated Writers have been renamed, previously they were called `{COMPONENT_NAME}.Requirable.Writer`, now they are called `{COMPONENT_NAME}Writer`.
```

Comment:
This flow a bit weird. Maybe:
The generated Writers have been renamed from `{COMPONENT_NAME}.Requirable.Writer` to `{COMPONENT_NAME}Writer`.

?

Reply:
Sure